nursery/chain.go

```diff
@@ -181,7 +181,45 @@ func (nursery *Nursery) handleChainSwapStatus(swap *database.ChainSwap, status b
 		}
 	}
 
+	var quoteError boltz.Error
 	switch parsedStatus {
+	case boltz.TransactionLockupFailed, boltz.TransactionMempool:
```

Comment:
```suggestion
	case boltz.TransactionLockupFailed:
```

Just lockup failed should be enough

Reply:
Amountless swaps dont go to lockup failed, so we need transaction mempool aswell (handles both cases)